packages/backend/src/plugins/catalog.ts

```diff
@@ -53,5 +53,15 @@ export default async function createPlugin(
   );
   unprocessed.registerRoutes();
   await processingEngine.start();
+
+  await env.scheduler.scheduleTask({
```

Comment:
I think it'd be better (more self-contained) to move this block to [here](https://github.com/manstis/backstage/blob/fa9ce70a5084fcbef971ef47516cc0baa72d457f/packages/backend/src/plugins/swf.ts#L23).

WDYT?

Reply:
agree, the more self-contained the better, wdyt to start this in the ServerlessWorkflowEntityProvider itself?

Reply:
@tiagodolphine Sure, why not :smile: 

You'll need to change [this](https://github.com/manstis/backstage/blob/fa9ce70a5084fcbef971ef47516cc0baa72d457f/packages/backend/src/plugins/catalog.ts#L39-L45) to pass the `scheduler: PluginTaskScheduler` too.